### PR TITLE
python3Packages.aioeagle: init at 1.1.0, python3Packages.ueagle: init at 0.0.2

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python library to control EAGLE-200

https://github.com/home-assistant-libs/aioeagle

Those are Home Assistant dependencies for > 2021.9.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 136936` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.ueagle</li>
    <li>python39Packages.ueagle</li>
  </ul>
</details>

**Review by samuela — CHANGES_REQUESTED** [review]
Why not separate into 2 PRs? Why isn't `aioeagle` added to python-packages.nix?

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -9076,6 +9076,8 @@ in {
 
   ufoprocessor = callPackage ../development/python-modules/ufoprocessor { };
 
+  ueagle = callPackage ../development/python-modules/ueagle { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:9079]: Missing aioeagle

**Review by SuperSandro2000 — DISMISSED** [review]

**Comment by fabaff** [comment]
> Why not separate into 2 PRs?

Would have been possible to split into two PR but as both will be required for Home Assistant I decided to keep them together.

If all requirements for an integration are combined then we can enable the tests in Home Assistant and update `component-packages.nix`. 

```nix
   "rainforest_eagle" = ps: with ps; [ ]; # missing inputs: aioeagle uEagle
```

This helps to minimize the merge conflicts and allow easier cherry-picking if needed.

> Why isn't aioeagle added to python-packages.nix?

That change is indeed missing.

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/top-level/python-packages.nix:9081]: Thanks, added.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 136936` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python38Packages.aioeagle</li>
    <li>python38Packages.ueagle</li>
    <li>python39Packages.aioeagle</li>
    <li>python39Packages.ueagle</li>
  </ul>
</details>

**Pull request merged**